learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,85 @@
-#  [Learning Bitmark / Quick start] - Simple solution for node setup
+#  [ Learning Bitmark / Quick start ] - Simple solution for node setup
+
+## Start your bitmark node
+
+A user can run a bitmark node in several ways. This section will guide you to start a bitmark node using docker container. Inside a bitmark node docker container, there are bitmarkd, recorderd, payment system and a user interface to control and monitor services.
+
+## Prerequisites
+
++ [Docker](https://docs.docker.com/install/) is installed in your system
+    +  Windows user run docker on Hyper-V
++ You [log in](https://docs.docker.com/engine/reference/commandline/login/) your docker with [docker hub]((https://hub.docker.com/)) account
++ You have and know your **public IP** address
++ You have **a bitcoin and a litecoin address** for receiving mining reward
++ Your environment is open for incoming connections through port 2136.  
+
+
+## Installation
+
+### Step 1: Download and Run Script
+
+Downloaded script will help you to manage bitmark node container. The  installation of the script will pull down a latest bitmark node container image from docker hub and then start a bitmark node container for you.  To check what functions the script supports and how to run it, simply run ``` bash install-node-linux-mac.sh```.  or ```bitmarkNode-HyperV.bat```.
+
+
++  Linux and mac users
+    + [Download scripts for linux and mac](https://bitmark-node-docker-scripts.s3-ap-northeast-1.amazonaws.com/install-node-linux-mac.sh)
+    + Execute bash install-node-linux-mac.sh [Your Public IP]. When run the script without providing [Your Public IP], the script will find your IP from Internet
+
+        ```bash install-node-linux-mac.sh  <Your Public IP>```
+    + Follow script instruction and select "1)Installation"
+
+    
++  Windows users
+    +  Ensure that Hyper-V is turned on
+    +  [Download script for windows](https://s3-ap-northeast-1.amazonaws.com/bitmark-node-docker-scripts/bitmarkNode-HyperV.bat)
+    + Execute bitmarkNode-HyperV.bat [Your Public IP]. When run the script without providing [Your Public IP], the script find your IP from Internet
+
+        ``` bitmarkNode-HyperV.bat 117.166.111.123```
+    + Follow script instruction and select "1)Installation"
++ Execute **docker ps** in your command-line/shell to check if bitmark-node is running
+![docker ps result](https://i.imgur.com/l3dF4Hl.jpg)
```

Comment:
And here, you probably want `docker ps` not **docker ps** for consistency. 

I think the style we've developed elsewhere is `command` or **App Menu**

Reply:
do you mean ```docker ps``` not  ***docker ps*** in markdown ? 
"I think the style we've developed elsewhere is command or App Menu" ? Do you mean we have a guideline or example?